lib/game.js

```diff
@@ -44,11 +47,16 @@ let appState = {
   ]
 };
 
-export default function initializeGame(emit) {
+export default function initializeGame(externalEmit) {
 
-  function handleSendCurrentState(uid, action) {
+  function emit(action) {
+    log('outbound: %o', action);
+    externalEmit(action);
+  }
+
+  function sendCurrentState(uid, action) {
     // TODO: put type in app state; character select mode / adventure mode
-    emit(Object.assign({ type: CHARACTER_SELECT_STATE }, appState));
+    emit(Object.assign({ type: appState.mode }, appState));
```

Comment:
it's a special case o use appState.mode as `type` on the client but as `mode` on the server. Can we keep it consistent, so the client `type` is `GAME_STATE_UPDATE` or something similar?